setup.py

```diff
@@ -370,7 +370,7 @@ def do_setup():
             'thrift>=0.9.2',
             'tzlocal>=1.4',
             'unicodecsv>=0.14.1',
-            'werkzeug>=0.14.1, <0.15.0',
+            'werkzeug>=0.15.0',
```

Comment:
I think let's just remove this entirely - we don't use werkzeug directly.

Reply:
Agree. It does not seem to be used